app/models/ext_management_system.rb

```diff
@@ -238,13 +238,15 @@ def validate_zone_not_maintenance_when_ems_enabled?
            :certificate_authority=,
            :to => :default_endpoint,
            :allow_nil => true
+  delegate :path, :path=, :to => :default_endpoint, :prefix => "endpoint", :allow_nil => true
 
   alias_method :address, :hostname # TODO: Remove all callers of address
 
   virtual_column :ipaddress,               :type => :string,  :uses => :endpoints
   virtual_column :hostname,                :type => :string,  :uses => :endpoints
   virtual_column :port,                    :type => :integer, :uses => :endpoints
   virtual_column :security_protocol,       :type => :string,  :uses => :endpoints
+  virtual_column :path,                    :type => :string,  :uses => :endpoints
```

Comment:
So this should be `:endpoint_path` now

Reply:
is the virtual_column necessary?

Reply:
Believe it is only needed if you want to query from the API, if you can submit the DDF form without it then should be okay to leave this out

Reply:
Done